.github/pull_request_template.md

```
@@ -0,0 +1,33 @@
+## What and why
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)